src/Mesh/babylon.mesh.ts

```diff
@@ -1621,79 +1621,162 @@
         }
 
         /**
-         * Normalize matrix weights so that all vertices have a total weight set to 1
+         *   Renormalize the mesh and patch it up if there are no weights
+         *   Similar to normalization by adding the weights comptue the reciprical and multiply all elements. this wil ensure that everything adds to 1. 

```

Comment:
Typos

Reply:
changed, and modified punctuation slightly